src/lib/y2storage/blk_device.rb

```diff
@@ -366,6 +385,21 @@ def multipath
       children.find { |dev| dev.is?(:multipath) }
     end
 
+    # Bcache device defined on top of the device, i.e. a Bcache device that uses
+    # this one as backing device
+    #
+    # @return [Bcache, nil] nil if the device is not part of any bcache
+    def bcache
```

Comment:
I'm fine with no defining the corresponding `direct_xxx` methods. They are not needed so far so they would only bring noise.

But to keep consistency with the current API in which `#filesystem`, `#lvm_pv` and `#md` return the corresponding device, no matter if there is a LUKS in the middle or not, this should do the same. I don't care how often we will have a LUKS between the real block device and its bcache, but its theoretically doable.

The current API may be far from perfection, but the worst API is an inconsistent one. :-)